packages/ptime/ptime.1.0.0+dune/opam

```diff
@@ -0,0 +1,44 @@
+opam-version: "2.0"
+maintainer: "Daniel Bünzli <daniel.buenzl i@erratique.ch>"
+authors: ["The ptime programmers"]
+homepage: "https://github.com/dune-universe/ptime"
+dev-repo: "git+https://github.com/dune-universe/ptime.git"
+bug-reports: "https://github.com/dbuenzli/ptime/issues"
+tags: [ "time" "posix" "system" "org:erratique" ]
+license: "ISC"
+depends: [
+  "dune"
+  "ocaml" {>= "4.01.0"}
```

Comment:
I'm confused, the description says it requires OCaml 4.08 (in the same vein, result is probably not required since it was added in 4.08).

Reply:
Ah yeah, that's the problem with rebasing older ports! I'll fix this!

Reply:
There were a few other things to fix in the opam file, it should be better now!